src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/AbstractRazorDelegatingEndpoint.cs

```diff
@@ -36,6 +36,8 @@ internal abstract class AbstractRazorDelegatingEndpoint<TRequest, TResponse> : I
         Logger = logger ?? throw new ArgumentNullException(nameof(logger));
     }
 
+    protected bool SingleServerSupport => _languageServerFeatureOptions.SingleServerSupport;
```

Comment:
How is SingleServerSupport different from OnlySingleServer?

Reply:
SingleServerSupport means "this client supports delegating requests to downstream language servers"

Normally these endpoints handle Razor contexts, and then if SingleServerSupport is on, they also can delegate to C# or HTML for those contexts. OnlySingleServer means "this endpoint should not run at all if SingleServerSupport isn't on". I think only one endpoint actually uses it, and I'm hoping to remove it in a future update to all of this stuff :D